sdk/communication/azure-communication-sms/azure/communication/sms/_shared/models.py

```diff
@@ -37,3 +44,103 @@ class UnknownIdentifier(object):
     """
     def __init__(self, identifier):
         self.identifier = identifier
+
+class CommunicationIdentifierModel(msrest.serialization.Model):
```

Comment:
Do we even need identifiers in /sms?

sms should use phone numbers as strings.

Reply:
Since python does not have the common package. we are trying to keep the _shared folder in sync across packages.